### PR TITLE
Adding market viewer tool with scripts to build and run it

### PR DESCRIPTION
**Pull request by peterbarrow**
Added a small tool to display the market depth information for a market.

### CODE REVIEW

**Review by jeremyletang — COMMENTED** [review]
any chances to add this as a subcommand of the package? It just feels like it's easier to use as you could just: `go install code.vegaprotocol.io/vegatools@latest` to access the whole suite of tools

File: marketdepthviewer/testnet.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./marketviewer -grpc=n09.testnet.vega.xyz:3002
```
Comment (jeremyletang) [inline marketdepthviewer/testnet.sh:2]: is this file needed?

File: marketdepthviewer/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (jeremyletang) [inline marketdepthviewer/build.sh:1]: I don't think this is needed as well?

**Review by jeremyletang — COMMENTED** [review]
It appears a bunch of file may not be needed + it seems the binary is commited in the marketdepthview folder.

File: marketdepthviewer/build.sh (hunk 2 above)
Comment (peterbarrow) [reply marketdepthviewer/build.sh]: Removed, thanks.

File: marketdepthviewer/testnet.sh (hunk 1 above)
Comment (peterbarrow) [reply marketdepthviewer/testnet.sh]: Removed, thanks.

File: marketviewer/marketviewer.go [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+	flags  *commandFlags
+	market *proto.Market
+)
+
+func processFlags() {
+	// Read any options from the command line
+	flag.StringVar(&flags.gRPCAddress, "grpc", "n08.testnet.vega.xyz:3002", "IP:Port of vega gRPC")
+	flag.StringVar(&flags.marketID, "market", "LBXRA65PN4FN5HBWRI2YBCOYDG2PBGYU", "MarketID")
+	flag.Parse()
+}
```
[the remark this answers is not in the document]
Comment (peterbarrow) [reply marketviewer/marketviewer.go]: I've updated the code to use the same conventions as the other commands.

File: marketviewer/marketviewer.go [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+	"code.vegaprotocol.io/vega/proto"
+	protoapi "code.vegaprotocol.io/vega/proto/api"
+	"github.com/gbin/goncurses"
+	"google.golang.org/grpc"
+)
+
+type commandFlags struct {
+	gRPCAddress string
+	marketID    string
+}
```
[the remark this answers is not in the document]
Comment (peterbarrow) [reply marketviewer/marketviewer.go]: Updated, thanks.
